server/stream.go

```diff
@@ -422,6 +429,9 @@ const (
 	JSResponseType            = "Nats-Response-Type"
 	JSMessageTTL              = "Nats-TTL"
 	JSMarkerReason            = "Nats-Marker-Reason"
+	JSBatchId                 = "Nats-Batch-Id"
+	JSBatchSeq                = "Nats-Batch-Sequence"
```

Comment:
Make it `Nats-Batch-Seq` I think

Reply:
Made it `Sequence` instead of `Seq`, because all other headers also have it fully. Like `Nats-Expected-Last-Sequence`.